internal/nginx-meshctl/commands/remove.go

```diff
@@ -106,10 +111,18 @@ func Remove() *cobra.Command {
 			}
 		}
 
-		resources, err := getProxiedResources(initK8sClient)
+		var resources proxiedResources
+		// the package functions that are called expect the controller-runtime client.
+		// ideally we can update our entire CLI to use this client instead of the current mismatched clients.
```

Comment:
Nit: capitalize the first word of the sentence to follow go's standard of using complete sentences for comments.